### PR TITLE
Adding rachel4n coordinates

### LINKED ISSUE #13112: Add me - London, UK
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by racheln4**
Hello 

@githubteacher have i done this correctly 

This is pull request closes #13112

### CODE REVIEW

**Comment by Evoltic** [comment]
@racheln4 great job! 🎉

File: _pins/racheln4.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: RACHELN4
+latitude: 51.5
+longitude: 0.3668
```
Comment (hectorsector) [inline _pins/racheln4.json]: Great work so far! You'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

Comment (racheln4) [reply _pins/racheln4.json]: Thank you so much!

**Comment by hectorsector** [comment]
:shipit: looks ✨ @racheln4

**Pull request merged**